Source/Documentation/Manual/options.rst

```diff
@@ -148,18 +148,18 @@ Other units
 -----------
 
 This selects the units displayed for length, mass, pressure, etc. in the
-:ref:`F5 HUD <driving-hud>` of the simulation.
+:ref:`F5 Train Driving Info Window and also the Alt+F5 HUD <driving-hud>` of the simulation.
 
 The option *Player's Location* sets the units according to the Windows
 *Language and Region* settings on the player's computer.
 
 The option *Route* sets the units based on the data in the route files.
 The other options are self-explanatory.
 
-The F5 HUD uses the abbreviations *stn* for short tons (2000 lb) and
-*t* or *tn* for metric tons (tonnes).
+These windows use the abbreviations *stn* for short tons (2000 lb),
+*t* for long tons (2,240 lb) or *tn* for metric tons (tonnes).
```

Comment:
Was the ton sign also changed in the code? The SI standard explicitly mentions that the metric ton sign is "t". So I would prefer having "t" for the metric, and e.g. "lt" for long ton.

Reply:
Hi Peter,

Thanks for catching this. The original text was wrong, but so was my correction. Guess I meant to check it and never did.

Now matches the code.

Best wishes,
Chris